com.eduke32.EDuke32.Plugin.DukeNukem3DShareware.yaml

```
@@ -0,0 +1,31 @@
+id: com.eduke32.EDuke32.Plugin.DukeNukem3DShareware
+branch: 1-3.32
+runtime: com.eduke32.EDuke32
+runtime-version: stable
+sdk: org.gnome.Sdk//3.32
```

```suggestion
sdk: org.gnome.Sdk
runtime-version: "3.32"
```
Please split this into sdk & runtime-version.

---

The original one was correct, your suggestion is not. It's an extension, so runtime here is the app, not Gnome runtime.

---

@gasinvein Oh my bad! I just saw this.  @FakeShemp Can you please revert this, please? Sorry for this.

---

@bilelmoussaoui That's alright, it's already replaced.